packages/web/docs/src/content/schema-registry/high-availability-cdn.mdx

```diff
@@ -11,11 +11,26 @@ GraphQL gateway, with [amazing uptime](https://www.cloudflare.com/business-sla/)
 Hive's status. This ensures that everything required for your GraphQL API and tooling around it is
 always available, and reduces the risk of depending on Hive as a single point of failure.
 
+## CDN Mirrors
+
+To provide an even higher level of availability, Hive offers two CDN endpoints:
+
+- `https://cdn.graphql-hive.com` (main): Built on top of Cloudflare, a fast and reliable global
+  network.
+- `https://cdn-mirror.graphql-hive.com` (mirror): An exact replica of the main CDN, built on top of
+  AWS (DNS, networking, and runtime). It serves as a fallback in the case of a Cloudflare outage.
+
+We recommend using the main CDN endpoint and configuring your systems to fall back to the mirror
+endpoint in case of any issues. Both endpoints are functionally identical, and share the same
+underlying storage and access tokens. Hive SDKs provide the option to configure a dual endpoint and
+have a built-in [circuit breaker](https://en.wikipedia.org/wiki/Circuit_breaker_design_pattern) that
+ensures a smooth experience in case the main CDN becomes unavailable.
+
```

Comment:
I think it can be useful to link to the implementation for more details 👍 
```suggestion

For more on configuring your Gateway and SDKs for high availability, see [Running with Hive Console in Production](/docs/schema-registry/high-availability-resilience).
```